src/pages/Encounters/tabs/EncounterNotesTab.tsx

```diff
@@ -715,9 +717,8 @@ export const EncounterNotesTab = ({
                             onKeyDown={(e) => {
                               if (e.key === "Enter" && e.shiftKey) {
                                 e.preventDefault();
-                                if (newMessage.trim()) {
-                                  handleSendMessage(e);
-                                }
+                                e.stopPropagation();
```

Comment:
can we stick to perform e.prevent / stop propogation either in handleSendMessage or here?

let's not call it multiple times here and there